common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -22,10 +22,31 @@ While the lowpass filters can effectively diminish the impact of this noise, hav
 
 For the gyro based rate controllers, this reduces their ability to respond to fast disturbances. If the gyro lowpass filter can be set higher, the phase lag induced is lower and the tune can be more aggressive. But this allows more noise and vibration, effectively canceling that gain out. Enabling either one or both of the Harmonic Notch filters allow targeting the noise generated by the motors, allowing a higher frequency of the following low pass to be set and therefore allowing tighter tune.
 
-The harmonic notch is enabled overall by setting :ref:`INS_HNTCH_ENABLE<INS_HNTCH_ENABLE>` = 1 for the first notch, and :ref:`INS_HNTC2_ENABLE<INS_HNTC2_ENABLE>` = 1, for the second. After rebooting, all the relevant parameters will appear.
+.. _notch_setup_overview:
+
+Notch Filter Setup Overview
+===========================
+
+#. Determine how the notch center frequency will be controlled. See :ref:`center_freq_control`.
+#. If a static notch (not usually recommended), or :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` control is used, the dominant noise frequencies will need to be determined in order to setup the notch. See :ref:`notch_center_freq`.
+#. :ref:`Enable the notch <notch_enable>`.
+#. Setup the selected center frequency control method using :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>`. And setup its associated parameters by reading its associated page linked in this secion :ref:`center_freq_control`.
```

Comment:
Can we start a new sentence with "And"? 
small typo, "secion" should be "section"